t/Test29-fregrid_no_stretched.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bats
+# test no stretched grid 
+
+@test "Test no stretched grid data lats 32.0 34.0 35.4" {
+
+  if [ ! -d "Test29" ] 
+  then
+  		mkdir Test29
+  fi
+
+  cd Test29
+  cp $top_srcdir/t/Test28-input/ocean_hgrid.nc . 
+  cp $top_srcdir/t/Test28-input/ocean_mosaic.nc .
+  cp $top_srcdir/t/Test28-input/topog.nc .
+
+#Make no stretched grid 
+  run command make_hgrid \
+               --grid_type gnomonic_ed \
+               --target_lon 262.4 \
```

Comment:
Do we need target lat/lon for non-stretched grid? What do they mean?

Reply:
Sorry to include target lat/lon for non-stretched grid I thought that it would play a role.